esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -1928,7 +1928,11 @@ static bool isUNCPath(const char* sprayPath, const char* dropZoneName, IProperty
         }
     }
     else if (!isHostInPlane(dropZone, hostInPath, true))
```

Comment:
only calling isHostInPlane for UNC (when dropzone) - when sourceIPReq is not supplied?

In dropzone case, if sourceIPReq is supplied, what checks the localPath of the UNC is within it?

Reply:
I think this block (and the if (!hostEp.isNull())) should be removed because the hostEp is always set.  If the sourcePlaneReq is empty, the sourceIPReq cannot be empty. If the sourcePlaneReq is not empty, the getDropZoneHost will be called with a possible sourceIPReq. In getDropZoneHost, if the sourceIPReq is empty, the sourceIPReq will be filled in. if the sourceIPReq is not empty, the sourceIPReq will be validated by isHostInPlane. Agree?